docs/modules/reference/pages/glossary.adoc

```diff
@@ -1,218 +1,247 @@
 
-JRobin:: A clone of RRDTool (see below) written in Java.
-Default time series database for new {page-component-title} instances.
+== K
 
-Kafka:: Clusterable, open-source message broker designed for high-throughput/low-latency event streaming.
-Originally developed at LinkedIn, it is now supported by the Apache community.
-(See xref:deployment:core/setup-message-broker.adoc[Set Up Message Broker].)
+https://kafka.apache.org/[Kafka]:: A clusterable, open-source message broker designed for high-throughput/low-latency event streaming.
+Originally developed at LinkedIn, it is now supported by the Apache community (see xref:deployment:core/setup-message-broker.adoc[]).
 
-Kibana:: A component of the Elastic Stack (comprised of Elasticsearch, Logstash, and Kibana).
-It provides visualization capabilities for data indexed in an Elasticsearch cluster.
+https://www.elastic.co/kibana/[Kibana]:: A component of the Elastic Stack (comprised of Elasticsearch, Logstash, and Kibana).
+It enables visualizations for data indexed in an Elasticsearch cluster.
 
-xref:operation:deep-dive/visualizations/opsboard/dashlets/ksc.adoc#ksc[KSC reports]:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data.
-They let you display data from different devices and sources (SNMP, ICMP, HTTP) on one page.
+KSC reports:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data.
+They let you display data from different devices and sources (SNMP, ICMP, HTTP) on one page (see xref:operation:deep-dive/visualizations/opsboard/dashlets/ksc.adoc[]).
 
 https://kubernetes.io/[Kubernetes]:: An open-source container orchestration system for automating software deployments, scaling, and management.
 Originally designed by Google, the Cloud Native Computing Foundation now maintains it.
 
-Location:: Also referred to as "monitoring location" or "Minion location".
-Nodes assigned to a monitoring location will be monitored by Minions assigned to the same location.
-This is distinct from geolocation, which is the actual physical location of the node.
-
-Management Information Base (MIB):: A hardware or software vendor file that describes SNMP objects their products provide.
-The MIB files can be processed to convert the definition into data collection and event objects for {page-component-title} to work with.
-Many of the major vendor MIB definitions are included out of the box.
+== M
 
 https://www.opennms.com/meridian/[Meridian]:: A subscription-based, optimized, and stable version of the OpenNMS Horizon platform that includes only the most stable and secure Horizon features.
 It is distributed via annual release with monthly security patches.
 
-Message broker:: When using Minions and Sentinels, a message broker is required for communication between servers.
+message broker:: When using Minions and Sentinels, a message broker is required for communication between servers.
```

Comment:
do we need to define what a message broker is, or can we assume users know that?

Reply:
```suggestion
message broker:: An asynchronous service that enables communication between an application's distributed components.
When using Minions and Sentinels, a message broker is required for communication between servers.
```